upgrades/upgrade.go

```diff
@@ -63,23 +63,6 @@ const (
 	DatabaseMaster = Target("databaseMaster")
 )
 
-// upgradeToVersion encapsulates the steps which need to be run to
-// upgrade any prior version of Juju to targetVersion.
-type upgradeToVersion struct {
-	targetVersion version.Number
-	steps         []Step
-}
-
-// Steps is defined on the Operation interface.
-func (u upgradeToVersion) Steps() []Step {
-	return u.steps
-}
-
-// TargetVersion is defined on the Operation interface.
-func (u upgradeToVersion) TargetVersion() version.Number {
-	return u.targetVersion
-}
-
```

Comment:
I don't think deleting these is best, they'll be needed in the future.

I believe in previous versions we have a 'hack' so that we can keep them unused, without compiler difficulties. Let me look

Reply:
I believe something like `var _ = upgradeToVersion{}` should work as well though. Just make sure to leave a comment explaining it's purpose

Happy for you to merge once you've resolved this your preferred way

Reply:
Added them back, looks like there is a `//nolint:unused` pragma